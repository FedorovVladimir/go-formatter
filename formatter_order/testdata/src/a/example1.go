package main

const c = "c" // want "formatter_order"

var v = "v" // want "formatter_order"

func a() { // want "formatter_order"
}
