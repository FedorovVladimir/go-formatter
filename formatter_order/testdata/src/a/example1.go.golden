package main

const c = "c"

var v = "v"

func a() {
}
